/*
 * This code was auto generated by AfterShip SDK Generator.
 * Do not edit the class manually.
 */
import { AfterShipErrorCodes } from "./code";

export const AfterShipMetaCodeMap: {[key: string]: AfterShipErrorCodes} = {
    "400": AfterShipErrorCodes.INVALID_REQUEST,
    "4001": AfterShipErrorCodes.INVALID_JSON,
    "4003": AfterShipErrorCodes.TRACKING_ALREADY_EXIST,
    "4004": AfterShipErrorCodes.TRACKING_DOES_NOT_EXIST,
    "4005": AfterShipErrorCodes.TRACKING_NUMBER_INVALID,
    "4006": AfterShipErrorCodes.TRACKING_REQUIRED,
    "4007": AfterShipErrorCodes.TRACKING_NUMBER_REQUIRED,
    "4008": AfterShipErrorCodes.VALUE_INVALID,
    "4009": AfterShipErrorCodes.VALUE_REQUIRED,
    "4010": AfterShipErrorCodes.SLUG_INVALID,
    "4011": AfterShipErrorCodes.MISSING_OR_INVALID_REQUIRED_FIELD,
    "4012": AfterShipErrorCodes.BAD_COURIER,
    "4013": AfterShipErrorCodes.INACTIVE_RETRACK_NOT_ALLOWED,
    "4014": AfterShipErrorCodes.NOTIFICATION_REUQIRED,
    "4015": AfterShipErrorCodes.ID_INVALID,
    "4016": AfterShipErrorCodes.RETRACK_ONCE_ALLOWED,
    "4017": AfterShipErrorCodes.TRACKING_NUMBER_FORMAT_INVALID,
    "401": AfterShipErrorCodes.API_KEY_INVALID,
    "403": AfterShipErrorCodes.REQUEST_NOT_ALLOWED,
    "404": AfterShipErrorCodes.NOT_FOUND,
    "429": AfterShipErrorCodes.TOO_MANY_REQUEST,
    "500": AfterShipErrorCodes.INTERNAL_ERROR,
    "502": AfterShipErrorCodes.INTERNAL_ERROR,
    "503": AfterShipErrorCodes.INTERNAL_ERROR,
    "504": AfterShipErrorCodes.INTERNAL_ERROR,
}